GOVERNANCE.md

```diff
@@ -2,94 +2,174 @@
 
 # Governance
 
-This document outlines the governance model for Astro. This includes the contributor model, code review, merging, and the consequences and process for Code of Conduct violations.
+This document outlines the governance model for Astro. This includes the contributor model, code review process, PR merge process, and the consequences of Code of Conduct violations.
 
-**All members must follow the [Code of Conduct](CODE_OF_CONDUCT.md).** Consequences for member violations are detailed in [Moderation](#moderation).
+👉 **All community members must follow the [Code of Conduct (CoC)](CODE_OF_CONDUCT.md).**  
+Consequences for CoC violations are detailed in [Moderation](#moderation).
+
+👉 **Want to trigger a vote, nomination, or perform some other action?**  
+Scroll down to [Playbook](#playbook).
 
 ## Get Involved
 
-Anyone can become an Astro Contributor regardless of skill level, experience, or background. All types of contribution are meaningful. Our membership system was designed to reflect this.
+**Anything that supports the Astro community is considered a contribution.** All types of contribution are meaningful, from code to documentation to blog posts. Anyone can become an Astro Contributor (yes, even you!). Our goal is to recognize all contributors to Astro regardless of skill level, experience or background.
+
+**@FredKSchott** wrote up some notes on the personal value of open source, which you can read in our [CONTRIBUTING.md](CONTRIBUTING.md#why-contribute-to-open-source) document.
+  
+## Contributor Levels
+
+We welcome people of all skill levels to become contributors. We recognize different degrees of contribution as **levels**, and most levels can be reached regardless of coding skill or years of experience. The two most important things that we look for in contributors are:
```

Comment:
Think this is a fine policy at our scale but I could see it being an issue as the project expands; should someone non-technical have a (potential) deciding vote on something technical? I think this is part of the reason why bigger projects have different "committees" that are in charge of different aspects of the project. Not requesting a change just noting something to keep in mind in the future.

Reply:
Note: In the Core Maintainer section code experiencing being needed is called out so maybe this part is ok after all.

Reply:
this was something that we actively wanted to address in these changes, where we have some great contributors and moderators in Discord not being properly recognized for their contributions in our governance model.